docs/en/patterns/navigation.md

```diff
@@ -5,40 +5,31 @@ table_of_contents: true
 
 ## Navigation
 
+<hr>
+
 Vanilla includes a simple navigation bar that you can add to the top of your
 sites.
 
 The navigation items are collapsed behind a "Menu" link in small screens and
 displayed horizontally on larger screens.
 
-Note: You can constrain the width of the navigation to match the
-`$grid-max-width` by placing everything inside the header element within a
-`.row` wrapper.
+<div class="p-notification--information">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Note:</span>You can constrain the width of the navigation to match the `$grid-max-width` by placing everything inside the header element within a `.row` wrapper.
```

Comment:
Should be `<code>$grid-max-width</code>`  because the markdown parser isn't converting it to code when it's part of an HTML block